File: pset1/mario_more.c
#include <stdio.h>
#include <string.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        printf("Height: ");
        height = get_int();

        if (height == 0)
        {
            return 0;
        }
    }
    while (height < 1 || height > 23);

    for (int row = 0; row < height; row++)
    {
        for (int i = 1; i < height - row; i++)
        {
            printf(" ");
        }

        for (int j = 0; j < row + 1; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int k = 0; k < row + 1; k++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6_2018/mario/mario_more.py
from cs50 import get_int


def main():
    height = prompt()
    build(height)    


def prompt():
    height = get_int("Height: ")
    if height < 0 or height > 23:
        return prompt()
    return height


def build(height):
    for row in range(height):
        [print(' ', end='') for x in range(1, height - row)]
        [print('#', end='') for n in range(row + 1)]
        print('  ', end='')
        [print('#', end='') for n in range(row + 1)]
        print()


if __name__ == '__main__':
    main()